data/blog/how-we-improved-performance-score-accuracy.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'How we improved Performance Score accuracy'
+date: '2024-01-17'
+tags: ['javascript', 'performance', 'web']
+draft: true
+summary: "We're making updates to how Performance Scores are calculated in the Web Vitals module, which will bring them closer to what your users experience."
+images: ['/images/how-we-improved-performance-score-accuracy/hero.jpg']
+layout: PostLayout
+canonicalUrl: performance-score-improvements
+authors: ['edwardgou']
+---
+
+Last year in November, we released the Web Vitals module for Sentry's Performance product. Aside from helping users monitor and improve their Web Vitals, this new module also introduced [Sentry Performance Scores](https://docs.sentry.io/product/performance/web-vitals/#performance-score) for all web browser applications. As a quick reminder, Performance Scores are used to condense multiple Web Vitals and their respective thresholds into an overall score from 0-100 that rates an app's perceived performance, based on real user data for Web Vitals.
+
+Today, we are making updates to how Performance Scores are calculated. These updates will improve the accuracy of these scores by bringing them closer to what users experience on your web apps. To understand how these changes will impact your Performance Scores, it's useful to understand how they used to be calculated. Here's a brief overview:
+
+1. Your users visit your webpage which generate pageload transactions. These transactions contain [Web Vitals](https://docs.sentry.io/product/performance/web-vitals/) (LCP, FCP, FID, TTFB, CLS) and get sent to Sentry.
+2. In the Sentry Web Vitals module, each web vital is aggregated to get the p75 value.
+3. The p75 value of each Web Vital is then put through a non-linear function which generates individual Web Vital component scores. Each component score is then statically weighted and summed to produce an overall Performance Score out of 100 for your web app. You can find out more about how these are calculate [here](https://docs.sentry.io/product/performance/web-vitals/#performance-score).
+
+While the above approach gives some of the signal that we're looking for, there are some drawbacks that affect how accurately our Performance Scores reflect what users experience:
+
+- Performance Scores can be skewed based on outlier pageloads if the Web Vitals are extreme enough.
+- Some Web Vitals may be reported less frequently, depending on [browser support](https://docs.sentry.io/product/performance/web-vitals/#browser-support). These Web Vitals end up overrepresented in Performance Scores because we use static weights.
```

Comment:
I think a quick example for each would be helpful! People might not be able to visualize what "static weights" are, especially since the equation isn't described above

Reply:
Updated to add more detail. I'm hoping with the equation rendering above, the weights concept will be more obvious